src/ext-test/java/org/opentripplanner/ext/siri/AddedTripBuilderTest.java

```diff
@@ -77,6 +78,7 @@ class AddedTripBuilderTest {
 
   private final Deduplicator DEDUPLICATOR = new Deduplicator();
   private final TransitModel TRANSIT_MODEL = new TransitModel(STOP_MODEL, DEDUPLICATOR);
+  private TransitEditorService TRANSIT_SERVICE;
```

Comment:
Should be `transitService`, it is not a constant.

Reply:
done